config/locales/zh-CN.yml

```diff
@@ -116,6 +116,7 @@ zh-CN:
       roles:
         admin: 管理员
         moderator: 协管
+        staff: 工作人员
```

Comment:
Obviously I have to change translations of `staff` because of this. Any other ideas?

Reply:
@Gargron Yep, I know, the original translation for `staff` is `管理员`, which is the same as `admin`. Don't think `工作人员` (staff) fits well here, so I'm asking for suggestions :P

Reply:
It's okay to use "admin" imo. Rather than separate between all/user/mod/admin this filter only separates between all and mod+admin, the results for mod+admin will not be numerous in most circumstances so it's okay to not subdivide them.